binding.gyp

```diff
@@ -18,6 +18,9 @@
           'sources': [
               'src/win32-dlfcn.cc'
           ],
+          'include_dirs': [
+            'deps/libffi/windows/<(target_arch)',
```

Comment:
How come you're adding this when `deps/libffi/config/win/<(target_arch)` is already in the include path?